superset/assets/src/dashboard/stylesheets/builder-sidepane.less

```diff
@@ -87,6 +87,7 @@
     }
 
     .card-title {
+      margin-right: 60px;
```

Comment:
here is to wrap title.

Reply:
@graceguo-supercat this is regardless of whether the slice has been added, right? I think we only want an early wrap if it has been added/the `ADDED` label is there?